metadata/Cargo.toml

```diff
@@ -16,7 +16,7 @@ description = "Command line utilities for checking metadata compatibility betwee
 codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive", "full"] }
 frame-metadata = "15.0.0"
 scale-info = "2.0.0"
-sp-core = "7.0.0"
+sp-core-hashing = "6.0.0"
```

Comment:
You could go one step further and use https://crates.io/crates/twox-hash directly here

Reply:
Ooh I remember now; I did do this initially but I had to copy a little code from sp-core-hashing to hash in the same way (offhand at least; I didn't dig into whether it could be simplified), and being aware that the hash functions (this and blake2) are needed in a couple of subxt crates I just ended up sticking with sp-core-hashing to provide them)